web/src/app/not-found.tsx

```diff
@@ -1,10 +1,12 @@
 import Link from "next/link";
 
-export default function NotFound() {
```

Comment:
will we not be using `function` at all or do we use it in certain situations?

Reply:
We can use it if that's the style convention we agree on. Let's go with `export default function NotFound()` and be consistent in this style for all our components.

Reply:
Personally, the convention I've used is if it's a component that returns something to be rendered, I use function, if it's a logic script (eg doing a fetch, generating a random code, etc) then I use const. What do you think?

Reply:
Sounds fine to me.